packages/integration-tests/src/test-project-setup/conversation_handler_project.ts

```diff
@@ -223,14 +209,24 @@ class ConversationHandlerTestProject extends TestProjectBase {
       backendId,
       authenticatedUserCredentials.accessToken,
       clientConfig.data.url,
-      apolloClient
+      apolloClient,
+      false
```

Comment:
Changes is in this file are to exercise same scenarios with and without streaming.